cmd/kubectl-directpv/drives.go

```diff
@@ -1,20 +1,18 @@
-/*
- * This file is part of MinIO Direct CSI
- * Copyright (C) 2021, MinIO, Inc.
- *
- * This code is free software: you can redistribute it and/or modify
- * it under the terms of the GNU Affero General Public License, version 3,
- * as published by the Free Software Foundation.
- *
- * This program is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
- * GNU Affero General Public License for more details.
- *
- * You should have received a copy of the GNU Affero General Public License, version 3,
- * along with this program.  If not, see <http://www.gnu.org/licenses/>
- *
- */
+// This file is part of MinIO DirectPV
```

Comment:
Just minimize the diff by changing respective lines. Do wherever applicable.

Reply:
This is on purpose Go's recommended way is to use `//` instead of `/*` for license header.

Almost all linters complain about this.

Reply:
Can you point me about the recommendation to know why `golangci-lint` workflow doesn't complain that?

Reply:
golangci-lint requires you to enable other linters that are not yet enabled @balamurugana Linters are a non exhaustive list 

That main reason is using `/*` conflicts with Go's build tags and their actual positions in the file.

This is why MinIO, Console and mc are all now using `//` instead of `/*` for license headers.

DirectPV should follow this as well.

Reply:
This should be done as separate PR along with enabling such linters in golangci-lint workflow. It is a bad idea reviewers to do linter work.

Reply:
No this PR is fine as is. We should take this PR as is. It fixes something valid.

Your argument is not valid here, I don't see why it should be done in separate PRs - it's not like it's fixing a bug.

Reply:
@balamurugana  I ran  `find . -name "*.go" -print0 | xargs --null addlicense -f "${SCRIPT_DIR}/boilerplate.go.txt"` . It adds comment as `//` . IMO we can take this PR as `/*` -> `//` just increases the diff nothing else , In many go files i saw `//` being used .

Reply:
@harshavardhana We should add specified change with enabled linter. If you want it in this PR, I am fine. This is only the way, multiple commented license header will be reported and fixed in upcoming PRs